content/webapp/pages/stories/index.tsx

```diff
@@ -51,11 +52,32 @@ const ArticlesContainer = styled.div`
   background-color: ${props => props.theme.color('warmNeutral.300')};
 `;
 
-const StoryPromoContainer = styled.div.attrs({
-  className: 'container container--scroll',
-})`
+const StoryPromoContainer = styled(Container)`
   -webkit-overflow-scrolling: touch;
 
+  /* former .container--scroll */
```

Comment:
AFAIK this is the only place it was used. I think @AMBacelar is working on adding it in Search as well, but I still feel it doesn't need to be a utility class. Could maybe be it's own component, or something else.